calypsoIframe.js

```diff
@@ -0,0 +1,26 @@
+document.addEventListener("DOMContentLoaded", function() {
```

Comment:
Should we add polyfills or a bundler such as webpack that takes care of the minification and all the dirty stuff (backwards compatibility).

Reply:
done